src/test/java/no/unit/nva/model/PublicationTest.java

```diff
@@ -115,6 +112,7 @@ private EntityDescription getEntityDescription() {
                 .withDate(getPublicationDate())
                 .withPublicationType(PublicationType.JOURNAL_ARTICLE)
                 .withContributors(Collections.singletonList(getContributor()))
+                .withAbstract("En lang streng som beskriver innholdet i dokumentet metdataene omtaler.")
```

Comment:
Not important, but shouldn't that be English?

Reply:
The publication language is set to Norwegian, so the abstract must match.